src/www/ui/template/admin_license_candidate.html.twig

```diff
@@ -50,7 +51,25 @@
     return otable;
   }
     
-    
+    function deleteCandidate(rfPk) {
+        if (confirm("Remove this license from candidate license list?")) {
+          $.ajax({
+            type: 'POST',
+            dataType: 'text',
+            url: '?mod=admin_license_candidate&action=deletecandidate',
+            data: { deleteRfPk : rfPk },
+            success: function(dataOnSuccess) {
+              if(dataOnSuccess == "true"){
+                $("#deletecandidate" + rfPk).closest("tr").hide();
+              }else{
+                alert(dataOnSuccess);
```

Comment:
we use `jquery-ui` which would allow you to use a more sophisticated alert dialog, like `$.alert(...)`.